ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml

```diff
@@ -25,10 +25,15 @@ releases:
 resources:
   '*':
     limits:
-      memory: 4Gi
+      memory: 12Gi
     requests:
       cpu: 100m
       memory: 200Mi
```

Comment:
@bipuladh Can we increasee these generic soft limits ? Tha should speed up the time spent by the plugin building container.